src/models/servercountrymodel.cpp

```diff
@@ -156,22 +158,14 @@ QVariant ServerCountryModel::data(const QModelIndex& index, int role) const {
 
     case CitiesRole: {
       const ServerCountry& country = m_countries.at(index.row());
-      qint64 now = QDateTime::currentSecsSinceEpoch();
 
       QList<QVariant> list;
       const QList<ServerCity>& cities = country.cities();
       for (const ServerCity& city : cities) {
-        int activeServerCount = 0;
-        for (QString pubkey : city.servers()) {
-          if (m_servers.value(pubkey).cooldownTimeout() <= now) {
-            activeServerCount++;
-          }
-        }
-
         QStringList names{
             city.name(),
             ServerI18N::translateCityName(country.code(), city.name()),
-            QString::number(activeServerCount)};
```

Comment:
We should expose an object instead of this QStringList

Reply:
I agree, this feels quite messy and it makes it quite error-prone to extend the city data.